examples/examples-cpu/nyc-taxi-snowflake/hyperparameter-dask.ipynb

```diff
@@ -20,6 +18,17 @@
     "</table>"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook describes a machine learning training workflow using the famous [NYC Taxi Dataset](https://www1.nyc.gov/site/tlc/about/tlc-trip-record-data.page). That dataset contains information on taxi trips in New York City.\n",
+    "\n",
+    "In this exercise, you'll load data into a [Dask DataFrame](https://docs.dask.org/en/latest/dataframe.html) and use [`dask-ml`](https://ml.dask.org/) to answer this question\n",
+    "\n",
+    "> based on characteristics that can be known at the beginning of a trip, will this trip result in a high tip?"
```

Comment:
wait sorry, I forgot that the examples-cpu examples are a regression problem. Will fix this soon

Reply:
ok fixed in https://github.com/saturncloud/examples/pull/68/commits/67defd4e712ecf94bcb01b9e4304e6d83f116d59. This is ready for review.